tests/ffa/test_flex_flash_attn.py

```
@@ -0,0 +1,420 @@
+
+def test_return_lse(monkeypatch):
+    def fake_ffa(**kwargs):
+        out = torch.full_like(kwargs["q"], 5)
+        lse = torch.full(
+            kwargs["q"].shape[:2], 11.0, dtype=torch.float32, device=kwargs["q"].device
+        )
+        return out, types.SimpleNamespace(lse=lse)
+
+    monkeypatch.setattr(ffa, "_load_flex_flash_attn_func", lambda: fake_ffa)
+
+    q_ranges, k_ranges, attn_type_map = _ranges()
```

Ruff (0.15.20)

[warning] Unpacked variable `attn_type_map` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)